torch/distributed/fsdp/_optim_utils.py

```diff
@@ -1341,13 +1353,11 @@ def _convert_all_state_info(
     input_states: Dict[str, Any],
     output_states: Dict[str, Dict[str, Any]],
 ) -> Tuple[Optional[torch.dtype], Dict[str, List[Optional[torch.Tensor]]]]:
-    """
-    Given the ``gathered_state_info`` and ``input_states``, the API converted
-    the StateInfo into the original state if the state is not a non-scalar
-    tensor. For a multi-dimensional tensor, the local state will be stored in
+    """Given the ``gathered_state_info`` and ``input_states``, the API converted the StateInfo into the original state if the state is not a non-scalar tensor. 
+
+    For a multi-dimensional tensor, the local state will be stored in
```

Comment:
To fix the "line too long" error:

```suggestion
    """
    Convert the the StateInfo into the original state.
    
    Given the ``gathered_state_info`` and ``input_states``, the API converts
    the StateInfo into the original state if the state is not a non-scalar tensor. 
    For a multi-dimensional tensor, the local state will be stored in
```